_episodes/01-introduction.md

```diff
@@ -1,15 +1,183 @@
 ---
-FIXME
+## Getting Started
+Python is a computer programming language that has become ubiquitous in scientific programming.  Our initial lessons will run python *interactively* through a python interpreter.  The Startup page should have provided information
+on how to start a python interpreter.  Everything included in a code block is something you could type into your python interpreter and evaluate.
 
-{% include links.md %}
+## Assigning variables
+
+Any python interpreter can work just like a calculator.  This is not very useful.
+
+```
+3+7
+```
+{: .language-python}
+
+```
+10
+```
+{: .output}
+
+You can assign values to variable names and then you can use those variables in subsequent commands.  
+```
+deltaH = -541.5   #kJ/mole
+deltaS =  10.4     #kJ/(mole K)
+temp = 298      #Kelvin
+deltaG = deltaH - temp*deltaS
+print(deltaG)
+```
+{: .language-python}
+
+```
+-3640.7000000000003
+```
+{: .output}
+
+Notice several things about this code.  The lines that begin with `#` are comment lines.  The computer does not do anything with these comments.  They have been used here to remind the user what units each of their values are in. Comments are also often used to explain what the code is doing, or leave information for future people who might use the code.
+
+In the previous code block, we also introduced the `print()` function.  Often, we will use the print function just to make sure our code is working correctly.
+
+Python can do what is called multiple assignment where you assign several variables their values on one line of code.  The following code block does the exact same thing as the previous code block.
+
+```
+#I can assign all these variables at once
+deltaH, deltaS, temp = -541.5, 10.4, 298
+deltaG = deltaH - temp*deltaS
+print(deltaG)
+```
+{: .language-python}
+
+```
+-3640.7000000000003
+```
+{: .output}
+
+Each variable is some particular type of data.  The most common types of data are strings, integers, and floats or floating point numbers.  In the code block below `label` is a string and `DeltaG` is a float.  It also shows how you can print more than one thing.
+```
+label = 'Delta G'    #this is a string
+print(label, ':', deltaG)
+```
+{: .language-python}
+
+```
+Delta G : -3640.7000000000003
+```
+{: .output}
+
+Another common data structure in python is the list.  Python has a built-in function `len` to determine the length of a list.
+```
+#This is a list; I can determine it's length
+energy_kcal = [-13.4, -2.7, 5.4, 42.1]
+length = len(energy_kcal)
+print('The length of this list is', length)
+```
+{: .language-python}
+
+```
+The length of this list is 4
+```
+{: .output}
+
+If I want to operate on a particular element of the list, you use the list name and then put in brackets which element of the list you want.  **In python counting starts at zero.  So the first element of the list is `list[0]`**
+
+```
+print(energy_kcal[1])
+```
+{: .language-python}
+
+```
+-2.7
+```
+{: .output}
 
```

Comment:
I think we may want to change the example here or add additional explanation. We want to show that you can perform operations on list elements. The first example (above) shows accessing the list element, while the next one shows reassigning the value. 

I think as a second demonstration, you could show multiplying this value, which would lead well into `for` loops. (What if we want to multiply all list values?)

The current second example seems to be a different concept to me (that concept is that you can assign list values by element, and that you can overwrite values in lists).